react/features/invite/components/InfoDialogButton.web.js

```diff
@@ -136,15 +122,10 @@ class InfoDialogButton extends Component {
         if (this.state.showDialog && !nextProps._toolboxVisible) {
             this._onDialogClose();
         }
-    }
 
-    /**
-     * Clear the timeout to automatically show the {@code InfoDialog}.
-     *
-     * @inheritdoc
-     */
-    componentWillUnmount() {
-        clearTimeout(this._autoShowTimeout);
+        if (!this.props._isConferenceJoined && nextProps._isConferenceJoined) {
```

Comment:
The function names (onDialogClose <-> maybeAutioShowDialog) seem to be opposites and this if seems to be disjunct with the previous one (including the conditions in the _maybeShowDialog function) so it's maybe worth merging them. Not a must have, it's just my OCD (either if or else :) )